build.gradle

```diff
@@ -28,24 +28,66 @@ ext {
 		!it.name.equals("spring-build-src") && !it.name.equals("spring-framework-bom")
 	}
 
-	aspectjVersion       = "1.9.2"
-	freemarkerVersion    = "2.3.28"
-	groovyVersion        = "2.5.5"
-	hsqldbVersion        = "2.4.1"
-	jackson2Version      = "2.9.8"
-	jettyVersion         = "9.4.14.v20181114"
-	junit5Version        = "5.3.2"
-	kotlinVersion        = "1.2.71"
-	log4jVersion         = "2.11.1"
-	nettyVersion         = "4.1.32.Final"
-	reactorVersion       = "Californium-SR4"
-	rxjavaVersion        = "1.3.8"
-	rxjavaAdapterVersion = "1.2.1"
-	rxjava2Version       = "2.2.5"
-	slf4jVersion         = "1.7.25"	  // spring-jcl + consistent 3rd party deps
-	tiles3Version        = "3.0.8"
-	tomcatVersion        = "9.0.14"
-	undertowVersion      = "2.0.16.Final"
+	aaltoXmlVersion           = "1.1.1"
+	activationApiVersion      = "1.2.0"
+	activationVersion         = "1.2.0"
+	annotationApiVersion      = "1.3.2"
+	aspectjVersion            = "1.9.2"
+	cacheApiVersion           = "1.1.0"
+	commonsFileuploadVersion  = "1.3.3"
+	commonsPoolVersion        = "2.6.0"
+	ejbApiVersion             = "3.2"
+	elApiVersion              = "3.0.1-b04"
```

Comment:
Which should be bofore? `elApiVersion`? `ejbApiVersion`?

_BTW: I sorted the lines with vim `sort`._